### PR TITLE
Fedimian City -> Fedimian Suburb

### PR DESCRIPTION
**Pull request by vtange**
- 1 - Note to IMC: On the "New Market District" Quest, the game tells you to find a boss at an incorrect location. 

The game tells you to find the boss "Moa" at "Abandoned Farm" at Fedimian Suburbs when the marker and the boss fight is actually at "Leipsna Chapel Site" at Fedimian Suburbs. I've changed the English text to direct players to the accurate location despite the Korean text.
- 2 -
  Also, there is an old man in Fedimian whose lines are broken up in 3. 

it used to be like:
QUEST_LV_0100_20150317_001207   There's an interesting stroy behind that gigantic statue.{nl}Should you have a chance
QUEST_LV_0100_20150317_001208   Silver of Fedimian
QUEST_LV_0100_20150317_001209   Read a book called

but that makes it so "Silver of Fedimian" shows up between line 1207 and line 1209.
so I've changed it for now like this, since English flows differently. 

QUEST_LV_0100_20150317_001207   There's an interesting story behind that gigantic statue.{nl}It's in a book called the  
QUEST_LV_0100_20150317_001208   Silver of Fedimian 
QUEST_LV_0100_20150317_001209   , you should read it if you get the chance.

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -3377,8 +3377,8 @@ ETC_20150317_003376	Wiping with scaffold
 ETC_20150317_003377	{memo X}Hook charcoal
 ETC_20150317_003378	It's not enough!
 ETC_20150317_003379	Amount charged is not enough!
-ETC_20150317_003380	Box Checking
-ETC_20150317_003381	Brother found the journal of brother!
+ETC_20150317_003380	$Checking the Box
+ETC_20150317_003381	Found the journal of the brother!
```
Comment (ttgmichael) [inline ETC.tsv]: Found Brother's Diary!

File: ETC.tsv [hunk 2]
```diff
@@ -3377,8 +3377,8 @@ ETC_20150317_003376	Wiping with scaffold
 ETC_20150317_003377	{memo X}Hook charcoal
 ETC_20150317_003378	It's not enough!
 ETC_20150317_003379	Amount charged is not enough!
-ETC_20150317_003380	Box Checking
-ETC_20150317_003381	Brother found the journal of brother!
+ETC_20150317_003380	$Checking the Box
```
Comment (ttgmichael) [inline ETC.tsv:3380]: Shouldn't 'box' be lower-cased?

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -638,21 +638,21 @@ QUEST_LV_0100_20150317_000637	$Venucelos are very brave and loyal.{nl}But, if th
...
-QUEST_LV_0100_20150317_000650	Yes! He will not end after stealing at the Mage Tower.{nl}I will go to Negybas Field first so follow me!
-QUEST_LV_0100_20150317_000651	You came. The next gravestone is in front of me, but I am just a drunken coward.{nl}I will stay at the road on the left side of Negybas Field so you go and check it.
-QUEST_LV_0100_20150317_000652	It is a shame to look back once a man starts his journey.{nl}My brother will rather choose to die than looking back.
-QUEST_LV_0100_20150317_000653	How could he think of using magic to the great magician?{nl}That's my brother.{nl}
-QUEST_LV_0100_20150317_000654	{memo X}We should check by rubbing Hallowventer's wooden coals.{nl}The next epitaph is at the right side of Negibas Fields.
-QUEST_LV_0100_20150317_000655	He will do anything if he decides to do something.{nl}How could we have imagined that he will climb onto Mage Tower which even demons failed to climb?
+QUEST_LV_0100_20150317_000641	$I am looking for my brother.{nl}If you can help me, I may give you some rewards.
+QUEST_LV_0100_20150317_000642	$He was a really fantastic person.{nl}A dream guy for people like me.
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: A dream guy for people like me. Sounds awkward.
Suggestion: I can't imagine myself being like him.

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -638,21 +638,21 @@ QUEST_LV_0100_20150317_000637	$Venucelos are very brave and loyal.{nl}But, if th
...
+QUEST_LV_0100_20150317_000643	$Something is between the pages of diary.{nl}A blueprint.. and oh, my brother had recorded a few things here.
+QUEST_LV_0100_20150317_000644	$I came here to look for my brother, but I am not as brave as he was.{nl}I want you to take a look at his gravestone's writing on my behalf.
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:644]: I want you to take a look at the writing on his gravestone on my behalf.

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -661,22 +661,22 @@ QUEST_LV_0100_20150317_000660	I am about to meet my brother.{nl}Please check the
...
-QUEST_LV_0100_20150317_000665	You should fight fire with fire.{nl}I will find the magician's weakness using magic and I will humiliate to her tower. Haha!
+QUEST_LV_0100_20150317_000664	$Then what does one do?{nl}
+QUEST_LV_0100_20150317_000665	$You have to fight fire with fire.{nl}I found the magician's weakness using magic and I humiliated her in her own tower. Haha!
 QUEST_LV_0100_20150317_000666	Stonemason Canolyn
-QUEST_LV_0100_20150317_000667	Are you going to Fedimian as well?{nl}We are stonemasons. We are called to rebuild Fedimian so w should hurry..{nl}
+QUEST_LV_0100_20150317_000667	$Are you going to Fedimian as well?{nl}We are stonemasons. We were called to rebuild Fedimian and were hurrying over there but..{nl}
 QUEST_LV_0100_20150317_000668	{memo X}
 QUEST_LV_0100_20150317_000669	There is no reason for those monsters walking around. {nl}They are just there.
-QUEST_LV_0100_20150317_000670	Hey, Why are you so late?{nl}While you were gone, monsters stole everything from us!
+QUEST_LV_0100_20150317_000670	$Hey, Why were you so late?{nl}While you were gone, the monsters came and stole everything from us!
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Why -> why

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -684,7 +684,7 @@ QUEST_LV_0100_20150317_000683	You got it. Thank you so much.{nl}You saved lots o
 QUEST_LV_0100_20150317_000684	{memo X}We can suppress the vicious energy if the magic of the tomb gets unleashed.{nl}I need the activating stone that is located inside the Guardians.
 QUEST_LV_0100_20150317_000685	The manual for the secret place
 QUEST_LV_0100_20150317_000686	$We hid the secret treasures for the Revelator who would come someday.{nl}Find and absorb the Guardians' hidden energy to open the secret door.
-QUEST_LV_0100_20150317_000687	The hiding place of the brother I used to serve is located at the abondoned farm up there.{nl}Please get the diary of the brother there.
+QUEST_LV_0100_20150317_000687	$The hideout of a brother I used to serve is located at the Abandoned Farm up north.{nl}Please get the diary of my brother there.
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: The hideout that I used to serve with my brother is located at the Abandoned Farm up north. ?

File: ETC.tsv (hunk 2 above)
Comment (Sunaries) [inline ETC.tsv:3380]: I like it this way. "Checking the Box" - I've done this myself too, for example: "Activating the Altar".

File: QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -638,21 +638,21 @@ QUEST_LV_0100_20150317_000637	$Venucelos are very brave and loyal.{nl}But, if th
...
+QUEST_LV_0100_20150317_000646	$So he challenged the Great Mage Tower?{nl}That's my brother! He can certainly do that.{nl}
+QUEST_LV_0100_20150317_000647	$The Mage Tower was a fortress that demons could not try to approach.
+QUEST_LV_0100_20150317_000648	$We should look for the next gravestone.{nl}If the writing can't be seen clearly, rub it with Cockatrice Fat.
+QUEST_LV_0100_20150317_000649	$Mage Tower.. Is there anything to steal there?
+QUEST_LV_0100_20150317_000650	$Yes! He will not end after stealing at the Mage Tower.{nl}I will go to Negyvas Field first, so follow me!
+QUEST_LV_0100_20150317_000651	$You came. The next gravestone is in front of me, but I am just a drunken coward.{nl}I will stay at the road on the left side of Negyvas Field so you go and check it.
+QUEST_LV_0100_20150317_000652	$It is a shame to look back once a man starts his journey.{nl}My brother would rather choose to die than look back.
+QUEST_LV_0100_20150317_000653	$How could he even think of using magic against the Great Wizard?{nl}That's my brother.{nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:653]: Great Wizard -> Grand Mage? (since there's a Grand Mage Tower instanced dungeon)

File: QUEST_LV_0100.tsv (hunk 6 above)
Comment (vtange) [inline QUEST_LV_0100.tsv]: Korean says this guy served his brother

File: QUEST_LV_0100.tsv (hunk 7 above)
Comment (vtange) [inline QUEST_LV_0100.tsv:653]: He challenges Ms. Flurry herself. She goes by "Great Wizard" in other lines.

File: QUEST.tsv [hunk 8]
```diff
@@ -2535,7 +2535,7 @@ QUEST_20151001_002542
 QUEST_20151001_002543	
 QUEST_20151001_002544	
 QUEST_20151001_002545	
-QUEST_20151001_002546	
+QUEST_20151001_002546	$Medis Diena was not Nature's will. After all, Nature is governed by the Goddess, and she would never do harm to us all.
```
Comment (Sunaries) [inline QUEST.tsv]: Why is nature capitalized?

**Comment by Sunaries** [comment]
Since Github mobile is bad, I'll leave this note in general. 

Ishirai Crossroads -> Ishrai Crossroads

File: QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -4810,66 +4810,66 @@ QUEST_LV_0100_20150317_004809	Collect demon souls
...
+QUEST_LV_0100_20150317_004847	$Defeat Cockatrices
+QUEST_LV_0100_20150317_004848	{memo X}Karoline Canolyn is worried about the Cockatrices that is blocking the way to Fedimian. Defeat Cockatrices that is blocking the way to Fedimian.
+QUEST_LV_0100_20150317_004849	$Report to Karoline Canolyn.
+QUEST_LV_0100_20150317_004850	$You defeated enough Cockatrices. Report to Karoline Canolyn.
+QUEST_LV_0100_20150317_004851	$Karoline Canolyn told you that the monsters stole all the tools. Talk to Karoline Canolyn again.
 QUEST_LV_0100_20150317_004852	Defeat Moa
-QUEST_LV_0100_20150317_004853	{memo X}Karoline Canolin told you that a monster called Moa stole the tool box. Defeat the moa that ran away to the upper side of the abondoned farm and retrieve the tool box.
-QUEST_LV_0100_20150317_004854	Unfortunately, the tool box wasn't there. Go back and talk to Karoline Canolin.
+QUEST_LV_0100_20150317_004853	{memo X}Karoline Canolyn told you that a monster called Moa stole the tool box. Defeat the Moa that ran away to the upper side of the Abandoned Farm and retrieve the tool box.
+QUEST_LV_0100_20150317_004854	$Unfortunately, the toolbox wasn't there. Go back and talk to Karoline Canolyn.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Go back -> Return (do this for the others as well)

(I've replaced a lot of "Go Back"s to "Return"s in my quest_lv_0100 PR.

File: QUEST.tsv (hunk 8 above)
Comment (vtange) [inline QUEST.tsv]: Just thought the Druid Master might refer to nature as an entity. Reverting it to lower case then.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile: 

1 - thanks for letting us know. :+1: 
2 - yup, that'll do

**Pull request merged**